CondFormats/MuonSystemAging/interface/MuonSystemAging.h

```diff
@@ -0,0 +1,42 @@
+#ifndef MuonSystemAging_H
+#define MuonSystemAging_H
+/*  example of polymorphic condition
+ *   *  LUT, function, mixed....
+ *    * this is just a prototype: classes do not need to be defined and declared in the same file
+ *     * at the moment though all derived classes better sit in the same package together with the base one
+ *      */
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include<cmath>
+#include<iostream>
+#include <vector>
+#include <regex>
+#include <map>
+
+enum CSCInefficiencyType 
+  { 
+    EFF_CHAMBER=0, 
+    EFF_STRIPS=1,
+    EFF_WIRES=2 
+  };
+
+class MuonSystemAging 
+{
+
+ public:
+
+  MuonSystemAging();
+  ~MuonSystemAging(){}
+
+  std::map<unsigned int, float>  m_RPCChambEffs;
```

Comment:
I find plenty of cases where a format using CON_SERIALIZABLE has maps in it by browsing the [release](https://github.com/cms-sw/cmssw/search?utf8=%E2%9C%93&q=COND_SERIALIZABLE+map&type=), am I missing something?